src/unix/fontutil.cpp

```diff
@@ -252,20 +258,28 @@ void wxNativeFontInfo::SetStyle(wxFontStyle style)
 
 void wxNativeFontInfo::SetWeight(wxFontWeight weight)
 {
-    switch (weight)
+    // deal with compatibility constants
```

Comment:
Would be nice to provide a single function doing this instead of repeating this (some day to be removed) code in multiple files. If we had a function, it would be easier to drop it, and all references to it, later.

Reply:
yes, agreed, I'll put something in fontcmn.cpp

Reply:
ok, done that, thanks for the feedback